components/javascript/toast.js

```diff
@@ -0,0 +1,26 @@
+const topToast = document.querySelector("#top-toast-id");
+const topToastContent = document.querySelector('#top-content');
+const topToastClose = document.querySelector('#top-toast-close');
+
+const bottomToast = document.querySelector('#bottom-toast-id');
+const bottomToastContent = document.querySelector('#bottom-content');
+const bottomToastClose = document.querySelector('#bottom-toast-close');
+
+const openTopToast = () => {
+    topToastContent.style.display = 'block';
+    setTimeout(() => closeTopToast(), 3000);
+}
+const closeTopToast = () => topToastContent.style.display = 'none';
+
+topToast.addEventListener("click", openTopToast);
+topToastClose.addEventListener("click", closeTopToast);
+
+
+const openBottomToast = () => {
+    bottomToastContent.style.display = 'block';
+    setTimeout(() => closeBottomToast(), 3000);
+}
+const closeBottomToast = () => bottomToastContent.style.display = 'none';
```

Comment:
Suggestion - instead of adding display block and none multiple times, you can add and

Reply:
created variables for block and none and used wherever necessary